.github/workflows/main.yaml

```diff
@@ -169,7 +174,9 @@ jobs:
         run: |
           docker load < /home/runner/build-tools/build-tools.tar
 
-      - run: make e2e
+      - run: |
+          export ENABLE_QUIC=${{ matrix.ENABLE_QUIC }}
+          make e2e
```

Comment:
maybe influence logs uploaded. both two e2e will use the same logs name: in line184